### PR TITLE
Unpack all downloading regex file

### PR DESCRIPTION
**Pull request by TheCoryBarker**
When downloading from Android Build based on regex, we need to return a list of all files downloaded, and unpack all of them

### CODE REVIEW

File: src/clusterfuzz/_internal/platforms/android/flash.py [hunk 1]
```diff
@@ -70,15 +70,18 @@ def download_latest_build(build_info, image_regexes, image_directory):
...
                      'branch %s and target %s.' %
-                     (image_file_path, build_info['branch'], target))
+                     (image_file_paths, build_info['branch'], target))
       return
-    if image_file_path.endswith('.zip') or image_file_path.endswith('.tar.gz'):
-      archive.unpack(image_file_path, image_directory)
+
+    for file_path in image_file_paths:
+      if file_path.endswith('.zip') or file_path.endswith('.tar.gz'):
+        archive.unpack(file_path, image_directory)
```
Comment (oliverchang) [inline src/clusterfuzz/_internal/platforms/android/flash.py:84]: Would these all be part of the same logical "build" ? i.e. is it ever possible these unpacks will clash/overwrite each other in a weird way?

**Review by oliverchang — APPROVED** [review]
LGTM with a question

**Comment by oliverchang** [comment]
Also, you need to fix format/lint

File: src/clusterfuzz/_internal/platforms/android/flash.py (hunk 1 above)
Comment (TheCoryBarker) [reply src/clusterfuzz/_internal/platforms/android/flash.py:84]: Yes these would all be part of the same logical build (i.e. Coverage enabled Cuttlefish build for x86 or HWASAN cuttlefish build for ARM). I would be surprised if there was clashing of files like boot.img but it seems possible

**Pull request merged**